src/ripple/app/ledger/impl/DisputedTx.h

```diff
@@ -36,22 +36,23 @@ namespace ripple {
 
     Undisputed transactions have no corresponding @ref DisputedTx object.
 */
+
+template <class Tx_t, class TxID_t, class NodeID_t>
```

Comment:
If this type participates in the templated consensus framework why not use use `Traits` instead of three individual template arguments?